docs/pql_examples.rst

```diff
@@ -148,6 +148,7 @@ Supported aggregations on single-value columns
 * ``MINMAXRANGE``
 * ``DISTINCTCOUNT``
 * ``DISTINCTCOUNTHLL``
+* ``DISTINCTCOUNTRAWHLL``: Returns serialized HLL after initial aggregation. Allows post-query aggregation by clients.
```

Comment:
You may wan to clarify that this is a string. Do you want to add example on how the result can be used?